app/javascript/packages/address-search/components/in-person-locations.tsx

```diff
@@ -15,14 +15,6 @@ export interface FormattedLocation {
   isPilot: boolean;
 }
 
-interface InPersonLocationsProps {
-  locations: FormattedLocation[] | null | undefined;
-  onSelect;
-  address: string;
-  noInPersonLocationsDisplay: ComponentType<{ address: string }>;
-  resultsHeaderComponent?: ComponentType;
-}
-
```

Comment:
Looks like we had the InPersonLocationsProps interface in both types.d and in in-person-locations.tsx. I deleted this one and updated the one in types.d to have the complete prop list.